scripts/scil_run_commit.py

```diff
@@ -0,0 +1,344 @@
+import nibabel as nib
+from nibabel.streamlines import Tractogram
+
+from scilpy.io.streamlines import (lazy_streamlines_count,
+                                   reconstruct_streamlines_from_hdf5)
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_processes_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_output_dirs_exist_and_empty)
+from scilpy.utils.bvec_bval_tools import fsl2mrtrix, identify_shells
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_tractogram',
+                   help='Input tractogram (.trk or .tck or .h5).')
+    p.add_argument('in_dwi',
+                   help='Diffusion from which the fodf were computed.')
+    p.add_argument('in_bvals',
+                   help='Bvals in the FSL format.')
+    p.add_argument('in_bvecs',
+                   help='Bvecs in the FSL format..')
+    p.add_argument('in_peaks',
+                   help='Peaks extracted from the fodf.')
+    p.add_argument('out_dir',
+                   help='Output directory for the COMMIT maps.')
+
+    p.add_argument('--tracking_mask',
+                   help='Binary mask were tratography was allowed.\n'
+                        'If not set, uses a binary mask computed from '
+                        'the streamlines.')
+
+    g1 = p.add_argument_group(title='Model options')
+    g1.add_argument('--ball_stick', action='store_true',
+                    help='Use the ball&Stick model.\n'
+                         'Disable the zeppelin compartment for single-shell.')
+    g1.add_argument('--parallel_diff', type=float,
+                    help='Parallel diffusivity in mm^2/s.\n'
+                    'Default for ball_stick: 1.7E-3\n'
+                    'Default for stick_zeppelin_ball: 1.7E-3')
+    g1.add_argument('--perpendicular_diff', nargs='+', type=float,
+                    help='Perpendicular diffusivity in mm^2/s.\n'
+                    'Default for ball_stick: None\n'
+                    'Default for stick_zeppelin_ball: '
+                    '[1.19E-3, 0.85E-3, 0.51E-3, 0.17E-3]')
+    g1.add_argument('--isotropic_diff', nargs='+', type=float,
+                    help='Istropic diffusivity in mm^2/s.\n'
+                    'Default for ball_stick: 2.0E-3\n'
+                    'Default for stick_zeppelin_ball: [1.7E-3, 3.0E-3]')
+
+    g2 = p.add_argument_group(title='Tractogram options')
+    g2.add_argument('--assign_weights', action='store_true',
+                    help='Store the streamlines weights in the '
+                    'data_per_streamline.')
+    g2.add_argument('--threshold_weights', type=float, metavar='THRESHOLD',
+                    help='Split the tractogram in two. Valid and invalid, '
+                    'based on the provided threshold.')
```

Comment:
Lets use other terms than valid and invalid. In fact, in the 'invalid' recobundle finds all bundles... so, it's not that the streamlines are invalid but not necessary to explain the signal. 

Not sure what the best terms are. 
'streamlines_needed' and 'unneeded_streamlines'...I'm sure we can do better